packages/fastify-htmx/index.js

```diff
@@ -108,7 +108,7 @@ async function prepareClient(clientModule, scope, config) {
 }
 
 // The return value of this function gets registered as reply.html()
-export function createHtmlFunction(source, scope, config) {
+export function createHtmlFunction(source, _, config) {
```

Comment:
I think I prefer leaving this unused argument name as `scope` just in case someone wants to use it in the future. I'm not sure if there's value in naming it `_`

Reply:
The problem with that convention is that it only accommodates for one optional parameter.

What we should really do is put all of these options into a single `ViteContext` interface.

But it can be done in the future, don't think it's a blocker for merging this PR straight away.